test/ex_doc/formatter/html_test.exs

```diff
@@ -100,12 +100,12 @@ defmodule ExDoc.Formatter.HTMLTest do
         generate_docs(doc_config(main: "Randomerror"))
       end)
 
-    assert output == "warning: index.html redirects to Randomerror.html, which does not exist\n"
+    assert output =~ "warning: index.html redirects to Randomerror.html, which does not exist\n"
     assert File.regular?("#{output_dir()}/index.html")
     assert File.regular?("#{output_dir()}/RandomError.html")
   end
 
-  test "warns on undefined functions" do
+  test "warns on undefined function within the same module" do
```

Comment:
not sure about this test description change, I don't think being within the same module is relevant, is it?

Reply:
nope. I have removed this in other PR i am submitting after this. I will delete it here.

Reply:
fixed  here.